### DISCUSSION TITLE
Safari 编辑器兼容性问题

### DISCUSSION THREAD

**Discussion by shiny** [geeknote-net/feedback#8] [General]
当中文输入法输入到一半，遇到行末换行时，会自动把中文输入法切换掉。
测试 Chrome 不存在该问题。
演示见附件

https://user-images.githubusercontent.com/117487/124606129-2a8d4600-de9f-11eb-976f-a4e1c659bdc2.mov

硬件：macbook pro m1 
浏览器：Safari
系统版本：11.4 (20F71)

**Comment by shiny** [comment]
这个问题好像应该发到 issue 区

**Comment by chloerei** [comment]
确实有问题，要花时间研究一下。

**Comment by chloerei** [comment]
已经修复了，有空可以试一试。
